test/sharness/t0030-ctl-pin.sh

```diff
@@ -0,0 +1,23 @@
+test_description="Test cluster-ctl's pinning and unpinning functionality"
+
+
+. lib/test-lib.sh
+
+
+test_expect_success "pin data to cluster with ctl" '
+    IPFS_PATH="../.test_ipfs" eval '"'"'ipfs add ../lib/sharness/sharness.sh >add_output.txt'"'"' &&
+    grep "added" add_output.txt &&
+    awk '"'"'{print $2}'"'"' add_output.txt > CID.txt &&
+    ipfs-cluster-ctl pin add $(< CID.txt) >pin_output.txt 
+    grep " PINNED" pin_output.txt >/dev/null &&
+    rm add_output.txt pin_output.txt
+'
```

Comment:
I think this can be expressed in simpler terms:

```
cid=$(echo "test" | ipfs add -q) &&
ipfs-cluster-ctl pin add "$cid" &&
ipfs-cluster-ctl pin ls "$cid" | grep -q "$cid" &&
ipfs-cluster-ctl status "$cid" | grep -q -i "PINNED"
```